source/usage-examples/find.txt

```diff
@@ -25,23 +25,20 @@ For more information on the ``sort()`` method, see our
 For more information on the ``projection()`` method, see our
 :doc:`guide on Projections </fundamentals/crud/read-operations/project/>`
 
-The ``find()`` method returns an instance of ``FindIterable``, a class
+The ``find()`` method returns an instance of ``FindFlow``, a class
 that offers several methods to access, organize, and traverse the results.
-``FindIterable`` also inherits methods from its parent class,
-``MongoIterable`` which implements the core Java interface ``Iterable``.
+``FindFlow`` also inherits methods from its parent class ``Flow`` from the
+Kotlin Coroutines library.
```

Comment:
Just FYI this may change due to https://www.mongodb.com/docs/spark-connector/master/configuration/read/#read-configuration

Reply:
I've commented this out for now with a note to update with accurate inheritance info once available post- https://jira.mongodb.org/browse/JAVA-4950